### ISSUE TITLE
Learning rate warmup with TF 

### ISSUE THREAD

**Issue by SoulGuedria** [uber/horovod#739]
Hi all, 

First of all thanks a lot for this nice project :) 
I'm trying to make the training of my CNN distributed and i noticed that even if i scale the learning rate with the number of GPUs, there is still a loss in the model accuracy using 12 GPUs.
So, i'm trying also to do the learning rate warmup but i can't find this functionality in the tensorflow version of hvd comparing to keras one because my CNN was originally implemented in TF.

I'm using Adam optimizer. Thanks a lot for your help.
